frequently-asked-questions.md

```diff
@@ -181,12 +188,15 @@ message.channel.send("Test").then(sentMessage => sentMessage.edit("Blah"));
 ```
 
 ```javascript
-// Fetching a message by ID (Discord.js versions 9 through 11)
+// Fetching a message by ID (Discord.js versions 9 through 13)
 // note: you can line return right before a "dot" in JS, that is valid.
 message.channel.messages.fetch({ around: "352292052538753025", limit: 1 })
   .then(messages => {
     const fetchedMsg = messages.first(); // messages is a collection!
-    // do something with it
+    // Check if the author of the message is the bot
+    if (message.client.user.id !== fetchedMsg.id) return console.log('I\'m not the author of that message!');
+
+    // Edit the message
     fetchedMsg.edit("This fetched message was edited");
   });
```

Comment:
Any reason in particular we are using around? We can fetch directly by id. Also no reason to mention this works from v9 up cause this guide isn't for v9 up.

Not sure if we should keep the `// note: you can line return right before a "dot" in JS, that is valid.` comment. This guide states: `We do, however expect you to have a basic understanding of JavaScript before attempting to write bots.` I believe this is covered by the basic js knowledge and this isn't the only place we have a newline with a dot after it.
```suggestion
// Fetching a message by ID
// note: you can line return right before a "dot" in JS, that is valid.
message.channel.messages.fetch("352292052538753025")
  .then(message => {
    // do something with it
    // Check if the author of the message is the bot
    if (message.client.user.id !== message.id) return console.log('I\'m not the author of that message!');
    // Edit the message
    message.edit("This fetched message was edited");
  });
  ```